ark/mibi/tiling_utils.py

```diff
@@ -382,27 +405,27 @@ def generate_x_y_fov_pairs(x_range, y_range):
     return all_pairs
 
 
-def create_tiled_regions(tiling_params, moly_point, tma=False):
-    """Create the tiled regions for each fov
+def generate_fov_list(tiling_params, moly_point, tma=False):
```

Comment:
Same here, I think we can remove the TMA logic

Reply:
Since we'll need to generate a FOV list in both the TMA and non-TMA case, I think we should have separate functions for both:

- `generate_fov_list` will become `generate_fov_list_non_tma` and basically be as is without the control-flow logic for TMAs.
- Create a new function `generate_fov_list_tma` which uses the two FOVs defined to define the centroids of each FOV. This function should take in user-input defining the number of x- and y-FOVs along each axis and verify that the FOVs contained in the file are valid. Using the user-defined params, generate the list of (x, y) pairs needed and associate a FOV with each.

Reply:
Yup, agreed. Given that there is no longer any logic about which parameter are required (it always be number of X and number of Y) probably it doesn't need to be an interactive prompt, it can just be two variables that are passed directly as arguments to the new function.